backend/game/engine/player_queue.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = []
+        self.dead_players = []
+        self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
```

Comment:
Why is this called add_initial_player? Are there going to be other kinds of players that could be added later on in the game?